.gitignore

```diff
@@ -68,3 +68,5 @@ coverage/
 # favicons - we want to ignore any downloaded .ico files but keep the directory that stores them.
 resources/favicons/*
 !resources/favicons/.gitkeep
+
```

Comment:
nit: remove empty line

Reply:
Thank you for pointing out. I removed the line.